go.mod

```diff
@@ -60,3 +60,5 @@ require (
 	k8s.io/kubernetes v1.13.3 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+replace github.com/kr/pty => github.com/creack/pty v1.1.7
```

Comment:
This is the key part of this PR (with the `Makefile` and the `presumbit-tests.sh` update)